src/EventStore/EventStore.ts

```diff
@@ -43,12 +50,12 @@ export const readEvents = async (es, eventId = 0) => {
 * @return {Promise<SOLIDITY_EVENT, Error>} json object representing the Solidity SOLIDITY_EVENT
 */
 export const writeEvent = async (es, transmuteEvent, fromAddress) => {
-  let meta = {
+let meta = {
     from: fromAddress,
     gas: 2000000
-  }
-  // console.log('writeSolidityEventAsync: ', transmuteEvent)
-  return await Middleware.writeSolidityEventAsync(es, meta, transmuteEvent)
+}
+// console.log('writeSolidityEventAsync: ', transmuteEvent)
```

Comment:
Please try not to leave debug statements in the code if you are done using them.